### ISSUE TITLE
"Request Fund"

### ISSUE THREAD

**Issue by Winthan** [interledgerjs/ilp-kit#165]
Will there be "Request Fund" function at ILP-kit? "Requesting Fund" could be fulfilled with latest exchange rates when sender approved.

**Comment by justmoon** [comment]
Yeah, this would be an SPSP feature. Unfortunately, there is a tricky spam/security issue. If we add such a feature, we have to figure out how to prevent people from spamming payment requests.

**Comment by Winthan** [comment]
@justmoon Yes, you are right. Usually, Paypal and Venmo is doing based on contact lists (Friend lists), as far as I see. The sender can accept or reject the request the payment, and until then the user has to wait for the respond.
